server/monitor/src/main/resources/resources/bulkImport.js

```diff
@@ -0,0 +1,213 @@
+   * create an empty array
+   */
+  var data = sessionStorage.bulkImports === undefined ?
+      [] : JSON.parse(sessionStorage.bulkImports);
+  var items = [];
+
+  /* If the data is empty, create an empty row, otherwise,
+   * create the rows for the table
+   */
+  if (data.length === 0 || data.bulkImport.length === 0) {
+    items.push('<td class="center" colspan="3"><i>Empty</i></td>');
+  } else {
+    $.each(data.bulkImport, function(key, val) {
+      items.push('<td class="firstcell left" data-value="' + val.filename +
+          '">' + val.filename + '</td>');
+
+      items.push('<td class="right" data-value="' + val.age + '">' + val.age +
+          '</td>');
+
+      items.push('<td class="right" data-value="' + val.state + '">' +
+          val.state + '</td>');
+    });
+  }
+
+  $('<tr/>', {
+    html: items.join('')
+  }).appendTo('#masterBulkImportStatus');
+}
+
+/**
+ * Generates the bulk import status table
+ */
+function refreshServerBulkTable() {
+
+  clearTable('bulkImportStatus');
+
+  /* Get the bulk import value obtained earlier, if it doesn't exists,
+   * create an empty array
+   */
+  var data = sessionStorage.bulkImports === undefined ?
+   [] : JSON.parse(sessionStorage.bulkImports);
+  var items = [];
+
+  /* If the data is empty, create an empty row, otherwise
+   * create the rows for the table
+   */
+  if (data.length === 0 || data.tabletServerBulkImport.length === 0) {
+    items.push('<td class="center" colspan="3"><i>Empty</i></td>');
+  } else {
+    $.each(data.tabletServerBulkImport, function(key, val) {
+      items.push('<td class="firstcell left" data-value="' + val.server +
+          '"><a href="/tservers?s=' + val.server + '">' + val.server +
+          '</a></td>');
+
+      items.push('<td class="right" data-value="' + val.importSize + '">' +
+          val.importSize + '</td>');
+
+      items.push('<td class="right" data-value="' + val.oldestAge + '">' +
+          (val.oldestAge > 0 ? val.oldestAge : '&mdash;') + '</td>');
+    });
+  }
+
+  $('<tr/>', {
+    html: items.join('')
+  }).appendTo('#bulkImportStatus');
+}
+
+/**
+ * Sorts the bulkImportStatus table on the selected column
+ *
+ * @param {string} table Table ID to sort
+ * @param {number} n Column number to sort by
+ */
+function sortTable(table, n) {
+  var tableIDs = ['bulkImportStatus', 'masterBulkImportStatus'];
+
+  if (sessionStorage.tableColumnSort !== undefined &&
+      sessionStorage.tableColumnSort == n &&
+      sessionStorage.direction !== undefined) {
+    direction = sessionStorage.direction === 'asc' ? 'desc' : 'asc';
+  } else {
+    direction = sessionStorage.direction === undefined ?
+        'asc' : sessionStorage.direction;
+  }
+  sessionStorage.tableColumn = tableIDs[table];
+  sessionStorage.tableColumnSort = n;
+  sortTables(tableIDs[table], direction, n);
+}
+
+/**
+ * Create tooltip for table column information
+ */
+$(function() {
+  $(document).tooltip();
```

Comment:
There wasn't (I just tested it and it works the same), I'll make the changes necessary on the files.